Code/GraphMol/FileParsers/v2_suppliers_catch.cpp

```diff
@@ -171,10 +171,10 @@ TEST_CASE("TestParsingInvalidChiralityLabelsWithMaeMolSupplier") {
       (boost::format(maeblock_template) % invalid_chirality_label).str();
 
   auto iss = std::make_unique<std::istringstream>(maeblock);
-  constexpr bool sanitize = false;
   constexpr bool takeOwnership = true;
+  constexpr FileParsers::MaeMolSupplierParams params{.sanitize = false};
```

Comment:
This syntax is weird, but definitely useful

Reply:
Agreed